src/dmd/dstruct.d

```diff
@@ -468,6 +468,13 @@ extern (C++) class StructDeclaration : AggregateDeclaration
             if (v.offset < offset)
             {
                 .error(loc, "overlapping initialization for `%s`", v.toChars());
+                string errorMsg = "struct initializers for `struct`s that contain anonymous `union`s";
+                errorMsg ~= " must initialize only the first member of a `union`. All subsequent";
+                errorMsg ~= " non-overlapping fields are default initialized";
```

Comment:
Just have this as a single chained concatenation.
```
string errorMsg = "struct initializers for `struct`s that contain anonymous `union`s" ~
	                " must initialize only the first member of a `union`. All subsequent" ~
	                " non-overlapping fields are default initialized";
```

Reply:
Thanks! Fixed.